fs/erofs/Config.in

```diff
@@ -6,10 +6,76 @@ config BR2_TARGET_ROOTFS_EROFS
 
 if BR2_TARGET_ROOTFS_EROFS
 
+choice
+	prompt "Compression algorithm"
+	default BR2_TARGET_ROOTFS_EROFS_LZ4HC
+	help
+	  Select the EROFS compression algorithm to use.
+
+config BR2_TARGET_ROOTFS_EROFS_NONE
+	bool "no compression"
+
+config BR2_TARGET_ROOTFS_EROFS_LZ4
+	bool "lz4"
+
 config BR2_TARGET_ROOTFS_EROFS_LZ4HC
-	bool "lz4hc compression"
+	bool "lz4hc"
+
+config BR2_TARGET_ROOTFS_EROFS_LZMA
```

Comment:
Not sure what is typically done in these cases, but maybe a depends or selects `BR2_PACKAGE_EROFS_UTILS_LZMA`?

Reply:
It's not desirable IMHO since you may not need `erofs-utils` at all in userspace. On the other hand you also may need appropriate kernel options but if I take other FS definitions for reference, it's not handled there either.